CONTRIBUTING.md

```diff
@@ -10,22 +10,20 @@ Before contributing to the `posit-sdk`, ensure that the following prerequisites
 
 - Python >=3.8
 
+## Instructions
+
+1. Fork the repository and clone it to your local machine.
```

Comment:
does this imply that all PRs should come from forks moving forward? Less a question about the correctness of this stance, more want to make sure @dbkegley and I follow the expected pattern as we're working on things.

Reply:
Ah, good call out! For posit-dev members, direct pull requests are welcome!